### PR TITLE
Add script file to automate training process

### PR DESCRIPTION
**Pull request by GopinathCool**

### CODE REVIEW

**Comment by lukas-blecher** [comment]
Thanks for your contribution.
I think it would be better to split the script up into the preparation phase and the training phase because you only have to download the data once but you might want to train multiple models.
Also, why is the the `--debug` flag set by default? To ignore wandb?

File: math_ocr_train.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+echo "Downloading datasets..."
+gdown https://drive.google.com/uc?id=176PKaCUDWmTJdQwc-OfkO0y8t4gLsIvQ
+gdown https://drive.google.com/uc?id=1QUjX6PFWPa-HBWdcY-7bA5TRVUnbyS1D
+gdown https://drive.google.com/uc?id=147uMjW5br-gbmAQ42Q1i0W3TbG7xS1hl
+echo "Extracting formulas..."
+unzip -q formulae.zip
+echo "Exraction done..."
+python3 dataset/dataset.py --equations ./math.txt --images ./train --tokenizer ./tokenizer.json --out ./dataset/data/train.pkl
+python3 dataset/dataset.py --equations ./math.txt --images ./test --tokenizer ./tokenizer.json --out ./dataset/data/test.pkl
+python3 dataset/dataset.py --equations ./math.txt --images ./val --tokenizer ./tokenizer.json --out ./dataset/data/val.pkl
```
Comment (lukas-blecher) [inline math_ocr_train.sh:17]: move to `setup.sh`

File: math_ocr_train.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+nohup python3 train.py --config ./settings/default.yaml --debug > train.log 2>&1 &
```
Comment (lukas-blecher) [inline math_ocr_train.sh:18]: why `--debug`?

Comment (GopinathCool) [reply math_ocr_train.sh:18]: Yes, by using wandb with CPU configuration, tremendously slow down the training, that's why I disabled it by default.

**Pull request closed without merging**